PasarGuardNodeBridge/controller.py

```
@@ -243,6 +244,46 @@ async def update_users(self, users: list[User]):
+            return None, None  # Already recovered
+        
+        try:
+            # Verify node is actually healthy before updating
+            await self.get_backend_stats()
+            await self.set_health(Health.HEALTHY)
+            health_status = "BROKEN" if was_broken else "INVALID"
+            self.logger.info(
+                f"[{self.name}] Sync succeeded while {health_status}, node health updated to HEALTHY"
+            )
+            # Return reset delays
+            return 10.0, 1.0
```

Ruff (0.14.5)

Consider moving this statement to an `else` block

(TRY300)